grafana/deploy.sh

```diff
@@ -0,0 +1,49 @@
+#!/bin/bash
+
+# This is a super simple approach to syncing dashboards
+# TODO:
+#  - check if dashboard really needs update (will cause many versions in grafana db)
+#  - check if stale folder needs to be deleted
+#  - check if stale dashboard needs to be deleted 
+
+set -eu
+
+export RESOURCEGROUP=${GLOBAL_RESOURCEGROUP:-global}
+
+if [[ -z ${GRAFANA_NAME} ]];
+then
+    echo "GRAFANA_NAME needs to be set"
+    exit 1
+fi
+
+cd dashboards
+
+existing_folders=$(az grafana folder list -g ${RESOURCEGROUP}  -n ${GRAFANA_NAME})
+
+IFS=$'\n'; for d in $(ls -1)
```

Comment:
This will trip if there is ever a file in the the directory

Maybe a more robust option:
```suggestion
for d in $(find . -mindepth 1 -maxdepth 1 -type d -exec basename {} \;)
```